packages/SwingSet/src/kernel/kernel.js

```diff
@@ -335,26 +334,34 @@ export default function buildKernel(
     doResolve(expectedDecider, [[kpid, true, errorData]]);
   }
 
-  function removeVatManager(vatID, shouldReject, info) {
-    insistCapData(info);
-    const old = ephemeral.vats.get(vatID);
-    assert(old, `no such vat: ${vatID}`);
-    ephemeral.vats.delete(vatID);
-    old.notifyTermination(shouldReject, info);
-    return old.manager.shutdown();
-  }
-
+  /*
+   * delete vat DB state,
+   * resolve orphaned promises,
+   * notify parent,
+   * shutdown worker
+   */
```

Comment:
What is the purpose of this comment?  It feels like a note to keep track of work while refactoring, but if so probably shouldn't be here -- either the work is done or it should be?

Reply:
It's a note from discussion with @warner ... but it does seem like useful documentation of what the function does. I suppose I can pretty it up a bit.

Reply:
cleaned up in c20652e